packages/engine/src/physics/systems/PhysicsSystem.ts

```diff
@@ -102,8 +102,6 @@ export default async function PhysicsSystem(
       const transform = getComponent(entity, TransformComponent)
       const network = getComponent(entity, NetworkObjectComponent)
 
-      if ((!isClient && network.userId === Engine.userId) || hasComponent(entity, AvatarComponent)) continue
```

Comment:
@speigg before we merge i want to make sure this wasn't doing anything specific i'm unaware of, but removing this line fixed the networked physics sync issue

Reply:
I don't remember why this check is here to be honest. Feel free to refactor as much as you think is necessary for proper state sync implementation!